pkg/lint/lintersdb/manager.go

```diff
@@ -247,6 +247,10 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		if stylecheckCfg != nil && stylecheckCfg.GoVersion != "" {
 			stylecheckCfg.GoVersion = trimGoVersion(m.cfg.Run.Go)
 		}
+
+		if goheaderCfg != nil {
+			goheaderCfg.LintConfigDir = m.cfg.GetConfigDir()
```

Comment:
I meant you can override `goheaderCfg.TemplatePath` at this level.
But I cannot decide which option is better